releases.json

```diff
@@ -1049,6 +1049,7 @@
       "godot_cpp"
     ],
     "versions": [
+      "4.3",
```

Comment:
```suggestion
      "4.3-1",
```

Reply:
Don't forget the revision (bumped if the wrap is updated, but the project version is unchanged).

Reply:
![image](https://github.com/user-attachments/assets/7d231b9c-cc88-4180-a099-9818b994714f)

I don't think that is correct :( For example 4.2 and 4.2-1 are different branches (and they point to different commits). I can write "4.3-0", if you think it would be better.

Reply:
It's the wrap revision, not the minor project version: `4.3-1` not `4.3.1`.

Reply:
Ok, got it.